.readthedocs.yaml

```diff
@@ -1,16 +1,29 @@
+# .readthedocs.yaml
 # Read the Docs configuration file
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
 # Required
 version: 2
 
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.8"
+
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
-   configuration: docs/source/conf.py
+  configuration: docs/source/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+  - pdf
```

Comment:
Does this now render to PDF? Do we want this?

Reply:
I remember only that it was in an example I was using as a basis for writing the config file. But I think you're right in that it's not *necessary*.

Having said that, I just went to the docs and clicked the PDF button that I never saw until you pointed it out... and tbh it's kinda cool 🤷‍♂️

Reply:
@StokesMIDE talking w/ Pete atm & trying to close up PR's. Can I consider this resolved?

Reply:
Go for it.